cli/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "code-push-cli",
-  "version": "1.10.0-beta",
+  "version": "1.10.1-beta",
```

Comment:
Let's make this `1.11.0-beta` since we're adding the new proxy feature, and also, don't forget to update the `code-push` dependency to `1.9.0-beta` to pull in the updates to the SDK needed for the proxy feature.